.github/workflows/BigIPReportLinting.yml

```diff
@@ -2,7 +2,7 @@ name: Lint our self-managed js files
 on:
   push:
     paths:
-      - 'underlay/js/bigipreport.js'
+      - 'underlay/js/bigipreport.ts'
       - '.eslintrc.json'
```

Comment:
Is this intentional? underlay should only contain js files, no?

Reply:
Since the code is now managed through typescript and the js files is the result of a transpiling process we're linting the source, aka, ts.

Reply:
But the source, the .ts file, is not in underlay/js/

Reply:
Nice catch!

Reply:
Adding that to my new PR